packages/app/navigation/linking.ts

```diff
@@ -0,0 +1,141 @@
+              },
+              GroupMembers: {
+                path: 'group/:groupId/members',
+                parse: parsePathParams('groupId'),
+              },
+              ManageChannels: {
+                path: 'group/:groupId/manage-channels',
+                parse: parsePathParams('groupId'),
+              },
+              Privacy: {
+                path: 'group/:groupId/privacy',
+                parse: parsePathParams('groupId'),
+              },
+              GroupRoles: {
+                path: 'group/:groupId/roles',
+                parse: parsePathParams('groupId'),
+              },
+            },
+          },
+          AppSettings: 'app-settings',
+          FeatureFlags: 'feature-flags',
+          ManageAccount: 'manage-account',
+          BlockedUsers: 'blocked-users',
+          WompWomp: 'report-bug',
+          AppInfo: 'app-info',
+          PushNotificationSettings: 'push-notification-settings',
+        },
+      },
+    },
+  },
+});
+
+export const getDesktopLinkingConfig = (
+  mode: string
+): LinkingOptions<DesktopBasePathStackParamList> => ({
+  prefixes: [],
+  config: {
+    screens: {
+      Root: {
+        path: basePathForMode(mode),
+        initialRouteName: 'Home',
+        screens: {
+          Activity: 'activity',
+          Profile: 'profile',
+          Home: {
+            screens: {
+              ChatList: '',
+              GroupChannels: 'group/:groupId',
+              Channel: {
+                initialRouteName: 'ChannelRoot',
+                path: 'group/:groupId/channel/:channelId',
+                parse: parsePathParams('channelId', 'groupId'),
+                screens: {
+                  ChannelRoot: {
+                    path: '',
+                  },
+                  UserProfile: {
+                    path: 'profile/:userId',
+                    parse: parsePathParams('userId'),
+                    exact: true,
+                  },
+                  EditProfile: {
+                    path: 'profile/:userId/edit',
+                    parse: parsePathParams('userId'),
+                    exact: true,
+                  },
+                  ChannelMembers: {
+                    path: 'group/:groupId/channel/:channelId/members',
+                    parse: parsePathParams('channelId'),
+                    exact: true,
+                  },
+                  ChannelMeta: {
+                    path: 'group/:groupId/channel/:channelId/meta',
+                    parse: parsePathParams('channelId'),
+                    exact: true,
+                  },
+                  GroupSettings: {},
+                  ChannelSearch: {},
+                  Post: {
+                    path: 'post/:postId',
+                  },
+                  ImageViewer: {},
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+});
+
+function parsePathParams(...keys: string[]) {
```

Comment:
Can we account for `/` characters here so they don't get URL encoded?

Reply:
I thought about this -- basically we'd choose a less ugly character that can never be part of a path to replace it with. I was thinking about a `+`? I didn't implement because I wasn't sure what the valid character set for flags actually is -- do you?

Reply:
Will leaving the slash intact mess up how it parses the route?

Flag is always `${ship}/${term}`, should be URL safe (https://docs.urbit.org/glossary/term)

Reply:
Yes, it unfortunately will. We could probably hack around, but I'd rather keep it simple for now -- gonna try with pluses.

Reply:
Went a little farther down this road, but found some funky bugs with customizing that behavior -- I think we should punt for now as it's mostly cosmetic.

Reply:
Got it, thanks for digging in. I'm okay punting